content/en/Platform Deep Dive/Collaboration/Organization/Organization Settings/SAML SSO/saml-migration.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "SAML Migration: Update Your Configuration"
+linkTitle: "Update SAML"
+weight: 30
+description: >
+  Learn how to update your SAML configuration from legacy to the new setup.
+---
+
+{{< pageinfo >}}
+<p>On April 24, 2023, we made some changes to our SAML implementation. To avoid any disruptions to authentication, update your SAML configuration.</p><p>During the transition period, your existing SAML SSO setup will be valid. After the transition period, we'll stop supporting legacy configurations.</p>
+{{< /pageinfo >}}
+
+This page is for [Organization Owners](/getting-started/glossary/#organization-owner) who have set up their SAML SSO configuration **before April 24, 2023**.
+
+- If you don't update your SAML configuration, users in your organization will have to authenticate with their email and password or using Google authentication (OAuth 2.0), if relevant.
+- Make sure to update configurations for each organization where you're an Organization Owner.
+- If you don't have a legacy SAML setup, please visit [Configure SAML SSO](/platform-deep-dive/collaboration/organization/organization-settings/saml-sso/) for instructions.
+
+## What Has Changed in the SAML Implementation
+
+Here is a summary of updates:
+
+- Our **Assertion Consumer Service (ACS) URL** is no longer `https://api.cobalt.io/users/saml/auth`, but is now unique per organization. To get your unique ACS URL, go to **Settings** > **Identity & Access** > **Configure SAML** in the Cobalt app.
+- We no longer require a **RelayState** within the assertion.
+- Organization Owners can now enable or disable [SSO enforcement](/platform-deep-dive/collaboration/organization/organization-settings/saml-sso/#enforce-saml-sso).
+
+## How to Update Your SAML Configuration
+
+There are two ways to migrate to the new SAML implementation:
+
+- **Option 1**: Create a new SAML app with your identity provider (IdP). For details, see our [instructions](/platform-deep-dive/collaboration/organization/organization-settings/saml-sso/#general-configuration-workflow).
+- **Option 2**: Update your existing SAML configuration between Cobalt and your IdP, as described below.
+
+Before updating your existing configuration, note the following:
+
+- To avoid disrupting anyone's workflows, choose a time when the impact on users is minimal.
```

Comment:
is it worth re-stating here they can also turn off the enforcement until they have tested the new configuration

Reply:
Added:

Turn off the [SAML enforcement](/platform-deep-dive/collaboration/organization/organization-settings/saml-sso/#enforce-saml-sso) until you’ve successfully tested your configuration.